test/test.getch.ts

```diff
@@ -449,11 +449,11 @@ describe('🎏 data handling', () => {
     assert.deepStrictEqual(res.data, {});
   });
 
-  it('replaces application/x-www-form-urlencoded with application/json', async () => {
+  it('application/x-www-form-urlencoded with object data should stringify with qs', async () => {
     const body = {hello: '🌎'};
     const scope = nock(url)
-      .matchHeader('Content-Type', 'application/json')
-      .post('/', JSON.stringify(body))
+      .matchHeader('Content-Type', 'application/x-www-form-urlencoded')
```

Comment:
This test was for the old behavior where `application/x-www-form-urlencoded` got magically rewritten as `application/json`.